DLRadioButton/DLRadioButton.m

```diff
@@ -75,11 +75,11 @@ - (void)setAnimationDuration:(CFTimeInterval)animationDuration {
 
 #pragma mark - Helpers
 
-- (void)drawButton {
-    if (!self.icon || [self.icon.accessibilityIdentifier isEqualToString:kGeneratedIconName]) {
+- (void)layoutSubviews {
+    if (!self.icon) {
         self.icon = [self drawIconWithSelection:NO];
     }
-    if (!self.iconSelected || [self.iconSelected.accessibilityIdentifier isEqualToString:kGeneratedIconName]) {
```

Comment:
I'm not actually sure about the removal of `[self.iconSelected.accessibilityIdentifier isEqualToString:kGeneratedIconName]` from the condition, but without it, it triggers infinite loop of layouts, at least when making a snapshot.